### DISCUSSION TITLE
can't understand the executable file in child_process section

### DISCUSSION THREAD

**Discussion by HomyeeKing** [nodejs/help#4899] [Q&A]
I wonder if the `.bat` and `.cmd` files are executable on Windows, I have searched a lot and the result seems confuse me

content from Node.js docs:
![image](https://user-images.githubusercontent.com/49113249/150309227-040a69ed-53f1-4a0d-8784-c42332770ac2.png)

the relevant search result:
https://aerorock.co.nz/list-of-executable-file-extensions-windows/

**Comment by aduh95** [comment]
When sharing a screenshot, please be mindful not everyone can read it, we should always provide alt text, and/or a link to the page where this screenshot was taken. For reference, the above image is a screenshot of this part of the docs: https://github.com/nodejs/node/blob/325b9473c03b2aaf28db44400a4093ed966a745b/doc/api/child_process.md#L93-L106
with emphasis on https://github.com/nodejs/node/blob/325b9473c03b2aaf28db44400a4093ed966a745b/doc/api/child_process.md#L99-L100

Do you have a suggestion on how to improve the wording if you think the current one is not accurate enough?
